advanced_samples/Bulk_Upload/DotNet/README.md

```diff
@@ -0,0 +1,59 @@
+# Bulk Uploader SDS DotNet Sample
+
+**Version:** 1.0.0
+
+[![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OCS/CSVtoOCS_DotNet?branchName=master)](https://dev.azure.com/osieng/engineering/_build?definitionId=1393&branchName=master)
+
+Developed against DotNet core 3.1.
+
+---
+
+## About this sample
+
+This sample uses SDS OCSclients to send values, streams, types and dataviews. This simple sample sends SDS JSON messages that are saved pre-formed as files named sdsType.json, sdsStream.json, sdsdata{streamID}.json, dataview.json. It sends the files in that order. If there are metadata or tag files it will send that for the streams.
+
+It does only basic error checking to make sure the message was accepted by the endpoint. The primary function of this sample is for easy bulk loading of data for other samples (particularly ML based samples where the amount of data is prohibitive to include in the sample itself). Included in
```

Comment:
Paragraph ends in a sentence fragment, not sure what you planned to say. "Included in"